app/models/conversion_host.rb

```diff
@@ -229,4 +230,10 @@ def tag_resource_as_disabled
   def default_name
     self.name ||= resource&.name
   end
+
+  # Set the default address to the first IP address associated with the resource.
+  #
+  def default_address
```

Comment:
`ipaddress` method already returns `address` if it is set, so I'm wondering what this method adds.

Reply:
If a user specifies an address explicitly via the rest-api for example, we want to use that first. This way it will, and only default to ipaddress if not specified.

Reply:
Then why not directly validate with `ipaddress` method ? It also returns `address` if it was passed on creation.

Reply:
Because that method doesn't do assignment.